docs/API-functions.md

```diff
@@ -917,6 +985,18 @@ it then blocks until at least one connection reports success or if all of them
 fail. Connections that continue in the background after this function exits can
 be then checked status by [`srt_group_data`](#srt_group_data).
 
+As member socket connections are running in the background, for getting known
+that particular connection has succeeded or failed it is recommended
+to use [`srt_connect_callback`](#srt_connect_callback). In this case the
+`token` callback function parameter will be the same as the `token` value used
+for particular item in the `name` connection table.
+
+The `token` value doesn't have any limitations except that -1 value is
+a "trap representation", that is, when set on input it will make the internals
+define a unique value for it. Your application can as well set unique values
+there by itself and in this case the token value will be preserved.
```

Comment:
```suggestion
The `token` value doesn't have any limitations except that the -1 value is
a "trap representation", that is, when set on input it will make the internals
define a unique value for the `token`. Your application can also set unique values,
in which case the `token` value will be preserved.
```